engine/engines.go

```diff
@@ -57,12 +57,15 @@ func Scrape(engine Engine) ([]Movie, error) {
 	)
 
 	// Add Cloud Flare scraper bypasser
-	if engine.getName() == "netnaija" {
-		t, err := scraper.NewTransport(http.DefaultTransport)
+	if engine.getName() == "NetNaija" {
+		log.Debug("Switching to Selenium transport")
+		t, err := transport.NewTransport(http.DefaultTransport)
 		if err != nil {
 			log.Fatal(err)
 		}
+
 		c.WithTransport(t)
+		c.SetCookieJar(t.Cookies)
```

Comment:
At this point the cookies are still empty. To pass colly the actual cookie You'll have to pass the colly pointer to transport and set it the cookies where you are done or pass the transport cookies pointer instead if that'll work